modules/rostests/kmtests/ntos_ob/ObQuery.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * PROJECT:     ReactOS kernel-mode tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Kernel mode tests for object information querying
+ * COPYRIGHT:   Copyright 2023 George Bișoc <george.bisoc@reactos.org>
+ */
+
+#include <kmt_test.h>
+
+#ifdef _WIN64
+#define OBJ_DIRECTORY_NAME_INFO_SIZE  34
```

Comment:
I find this a little confusingly named since it's the size for the "Windows" directory specifically, not for any directory.
(I'd still find `sizeof(UNICODE_STRING) + sizeof(L"\\Windows")` most useful, but just numbers work if you prefer 🤷)

Reply:
The sizeof solution would indeed be more portable, and this would remove the question "why this value 34 and not e.g. 42 or sthg.".

Reply:
I refactored the defines, opting for a "sizeof" approach instead. Let me know if this is OK for you.